modules/JsCarV2G/index.js

```diff
@@ -66,9 +66,24 @@ function JavaStartedDeferred(mqtt_base_path, module_name, mod) {
   });
 }
 
+const net_init = os.networkInterfaces();
+
+function choose_first_network_iface() {
+  for (let iface in net_init) {
+    for (let element of net_init[iface]) {
+      if (element.hasOwnProperty("address")) {
+        if (element.address.includes('fe80')) {
+          return iface
+        }
+      }
+    }
+  }
+  evlog.warning(`No necessary IPv6 link-local address was found!`)
+  return "eth0"
+}
+
```

Comment:
Quite some codacy issues, here.  We need to get some setup for the js linting. There is already an .eslint.rc in the root folder of everest core.  Nevertheless, the following code would be more js-like, I guess:
```
function choose_first_ipv6_local() {
  const all_ifs = os.networkInterfaces();

  const found_if = Object.keys(all_ifs).find(
    (iface) => all_ifs[iface].filter((info) => info.address.includes('fe80')).length > 0
  );

  if (found_if) return found_if;

  evlog.warning('No necessary IPv6 link-local address was found!');
  return 'eth0';
}
```

Reply:
Tried your suggestion. It is much better.